edge-util/src/Microsoft.Azure.Devices.Edge.Storage.RocksDb/disk/ThresholdDiskSpaceChecker.cs

```diff
@@ -14,7 +14,7 @@ class ThresholdDiskSpaceChecker : DiskSpaceCheckerBase
         public ThresholdDiskSpaceChecker(string drive, double thresholdPercentage, TimeSpan checkFrequency, ILogger logger)
             : base(checkFrequency, logger)
         {
-            Preconditions.CheckArgument(thresholdPercentage >= 0 && thresholdPercentage <= 100, $"Invalid thresholdPercentage value {thresholdPercentage}");
+            Preconditions.CheckArgument(thresholdPercentage >= 0 && thresholdPercentage <= 1, $"Invalid thresholdPercentage value {thresholdPercentage}");
```

Comment:
So just to clarify, the comment I had made about percentages vs ratios was specifically for the `double usagePercentage = percentDiskUsed * 100 / thresholdPercentage;` calculation, since that was calculating a percentage of percentages. For `thresholdPercentage` and `percentDiskUsed`, keeping them as percentage is fine.

If you do want to change them to ratios too, then you should rename the variables to not have `percent` or `percentage` in the name. But I think it'd be better to keep them as percentages in the first place.